os/drivers/input/touchscreen.c

```diff
@@ -512,7 +515,14 @@ int touch_register(const char *path, struct touchscreen_s *dev)
 	}
 	touch_semgive(&dev->pollsem);
 #endif
-	int ret = register_driver(path, &g_touchdev_fileops, 0666, dev);
+	silent_reboot_is_silent_mode(&is_silent_mode);
```

Comment:
I changed type of this function. Please check below commit and update it.
https://github.com/Samsung/TizenRT/commit/3007c01245b3266dc92c3063faaaad2d51a3dec4
```
bool is_silent_mode;
is_silent_mode = silent_reboot_is_silent_mode();
```